certified-connectors/Viafirma/apiDefinition.swagger.json

```diff
@@ -0,0 +1,341 @@
+            "type": "string",
+            "default": "*/*\r",
+            "description": "Accept",
+            "x-ms-summary": "Accept"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "groupCode": {
+                  "type": "string",
+                  "description": "groupCode",
+                  "title": "",
+                  "x-ms-dynamic-values": {
+                    "operationId": "WhoIAm",
+                    "value-path": "groups-key-item-output",
+                    "value-collection": "groups",
+                    "value-title": "groups-key-item-output"
+                  }
+                },
+                "workflow": {
+                  "type": "object",
+                  "properties": {
+                    "type": {
+                      "type": "string",
+                      "description": "type",
+                      "title": "",
+                      "default": "PRESENTIAL",
+                      "x-ms-visibility": "internal"
+                    }
+                  },
+                  "description": "workflow",
+                  "required": [
+                    "type"
+                  ]
+                },
+                "document": {
+                  "type": "object",
+                  "properties": {
+                    "templateCode": {
+                      "type": "string",
+                      "description": "templateCode",
+                      "title": ""
+                    }
+                  },
+                  "description": "document",
+                  "required": [
+                    "templateCode"
+                  ]
+                }
+              },
+              "default": {
+                "groupCode": "your_group_here",
+                "workflow": {
+                  "type": "WEB"
+                },
+                "document": {
+                  "templateCode": "templateSample"
+                }
+              },
+              "required": [
+                "workflow",
+                "document",
+                "groupCode"
+              ]
+            },
+            "required": true
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "200",
+            "schema": {}
+          },
+          "default": {
+            "description": "default"
+          }
+        },
+        "x-ms-visibility": "important"
+      }
+    },
+    "/documents/api/v3/system/alive": {},
+    "/documents/api/v3//template/list/mine": {},
+    "/documents/api/v3/template/list/mine": {
+      "get": {
+        "summary": "Get user's templates",
+        "description": "List all the user's templates",
+        "operationId": "GetUserTemplates",
+        "parameters": [],
+        "responses": {
+          "200": {
+            "description": "200",
+            "schema": {}
```

Comment:
@anfuca Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.